.gitlab-ci.yml

```diff
@@ -6,6 +6,8 @@ stages:
 
 variables:
   EXPORT_NAME: game-off
+  ITCHIO_USERNAME: user
+  ITCHIO_GAME: game
 
```

Comment:
Since itch.io is optional i think these variables should be included in the gitlab ci panel

Reply:
Good idea. I fixed it in the last commit and updated the documentation